### PR TITLE
App-template 

### PR DESCRIPTION
**Pull request by aperacaula**
Landing, login and register completed
Profile and home are there but still to improve

### CODE REVIEW

File: staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function toCamelCase(str) {
```
Comment (manuelbarzi) [inline staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js:2]: TIP 'use strict';

File: staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  if (typeof str === "string") {
+    var arr = str.toLowerCase().split(" ");
+    for (var i = 1; i < arr.length; i++) {
+      arr[i] = arr[i][0].toUpperCase() + arr[i].slice(1);
+    }
+    return arr.join("");
+  }else{
+      throw Error('input is not a string')
```
Comment (manuelbarzi) [inline staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js]: TIP ; at the end of statement

File: staff/aperacaula/js/challenges/count-words/scripts/main.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+var count= countWords('hello world');
```
Comment (manuelbarzi) [inline staff/aperacaula/js/challenges/count-words/scripts/main.js:2]: TIP 'use strict';

File: staff/aperacaula/js/challenges/count-words/scripts/countWords.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+function countWords(str){
```
Comment (manuelbarzi) [inline staff/aperacaula/js/challenges/count-words/scripts/countWords.js:2]: TIP 'use strict';

File: staff/aperacaula/js/challenges/count-chars/scripts/main.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+var count= countChars('hello world');
```
Comment (manuelbarzi) [inline staff/aperacaula/js/challenges/count-chars/scripts/main.js:2]: TIP 'use strict';

File: staff/aperacaula/js/challenges/count-chars/scripts/countChars.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+function countChars(str){
```
Comment (manuelbarzi) [inline staff/aperacaula/js/challenges/count-chars/scripts/countChars.js:2]: TIP 'use strict';

File: staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js (hunk 1 above)
Comment (aperacaula) [reply staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js:2]: Ok, done everywhere! Thanks

File: staff/aperacaula/js/challenges/count-words/scripts/countWords.js (hunk 4 above)
Comment (aperacaula) [reply staff/aperacaula/js/challenges/count-words/scripts/countWords.js:2]: 👌🏻

File: staff/aperacaula/js/challenges/count-chars/scripts/countChars.js (hunk 6 above)
Comment (aperacaula) [reply staff/aperacaula/js/challenges/count-chars/scripts/countChars.js:2]: done!!

File: staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js (hunk 2 above)
Comment (aperacaula) [reply staff/aperacaula/js/challenges/to-camel-case/scripts/toCamelCase.js]: alright!

**Pull request merged**